### PR TITLE
ENH: Support to export file tags and zip_member_count 

### PR DESCRIPTION
**Pull request by qian587**
Resolves GEAR-1224 and GEAR-1559

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -2,6 +2,6 @@ pydicom~=2.0.0
 pytest
 pytest-cov
 pytest-mock
-flywheel-sdk>=11.0.0
+flywheel-sdk==12.4.0
```
Comment (NPann) [inline tests/requirements.txt:5]: why not the same?

**Review by NPann — APPROVED** [review]
LGTM. 

Do you think you could move that one to gitlab flywheel-apps/genentech and archive this repo as we did for GRP-13 ?

File: tests/requirements.txt (hunk 1 above)
Comment (qian587) [reply tests/requirements.txt:5]: because of the geartoolkit version is dependent on that specific version. Should I bump up the gtk version here?

Comment (NPann) [reply tests/requirements.txt:5]: Let's just remove it then. Will be install by the gear-toolkit

**Comment by qian587** [comment]
Closing this MR as will be moving this into gitlab

**Pull request closed without merging**